core/comments/workspace_comment.ts

```diff
@@ -56,7 +57,21 @@ export class WorkspaceComment {
     // TODO: File an issue to remove this once everything is migrated.
     workspace.addTopComment(this as AnyDuringMigration);
 
-    // TODO(7909): Fire events.
+    this.fireCreateEvent();
+  }
+
+  private fireCreateEvent() {
+    if (eventUtils.isEnabled()) {
+      // TODO: Before merging. In the old class, this is wrapped by a setGroup
```

Comment:
@NeilFraser could you check this for me? Wanted a second set of eyes.

Reply:
I don't see any point to it. It would be a group with definitely only one event in it.

The other places where we use groups, there is at least the possibility of multiple events being fired in between where we set true and false. Here, I don't see any such possibility. So I think it's fine to remove.